roles/uchiwa/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+ - firewall
+ - sensu/server
```

Comment:
Sure, but I figured we might as well design things with that in mind unless there is a compelling reason to *limit* deployment options right now.